IoTEdgeModbus/Modules/AzureIoTEdgeModbus/Slave/SessionsHandle.cs

```diff
@@ -68,20 +68,20 @@ public void Release()
             }
             this.ModbusSessionList.Clear();
         }
-        public List<object> CollectAndResetOutMessageFromSessions()
+        public List<ModbusOutContent> CollectAndResetOutMessageFromSessions()
         {
-            List<object> obj_list = new List<object>();
+            var contents = new List<ModbusOutContent>();
 
             foreach (ModbusSlaveSession session in this.ModbusSessionList)
             {
-                var obj = session.GetOutMessage();
-                if (obj != null)
+                ModbusOutContent message = session.GetOutMessage();
```

Comment:
Please use var instead of full type name where possible as a coding convention. Not an issue as such :).
So-
ModbusOutContent message = session.GetOutMessage(); -> var message = session.GetOutMessage();